src/containers/RecruiterDashboard/RecruiterJobPostDetails.jsx

```diff
@@ -0,0 +1,193 @@
+import { React, useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import {
+  Link,
+  Card,
+  CardContent,
+  Grid,
+  Button,
+  Avatar,
+  TextField,
+  TextareaAutosize,
+  Box,
+  Typography,
+  makeStyles,
+} from "@material-ui/core";
+
+import { SummaryComponent, Emailer } from "../../common";
+//import recruiterData from "./recruiterData";
+
+import {
+  jobs,
+  jobApplicantsForPost,
+  candidateStatus,
+  users,
+  ashwekPawar,
+  recruiter,
+  allCities,
+} from "./recruiterData";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    marginLeft: "auto",
+    marginRight: "auto",
+    marginTop: theme.spacing(10),
+    marginBottom: theme.spacing(2),
+    padding: theme.spacing(2),
+    maxWidth: "1000px",
+    height: "80%",
+  },
+  title: {
+    fontWeight: "900",
+  },
+  skill: {
+    backgroundColor: "#EDEDED",
+    borderRadius: "6px",
+    margin: theme.spacing(2),
+  },
+  bold: {
+    fontWeight: "900",
+  },
+  avatar: {
+    width: theme.spacing(10),
+    height: theme.spacing(10),
+    margin: theme.spacing(2),
+  },
+  viewButton: {
+    backgroundColor: "#02bfa0",
+    color: "#fff",
+    textAlign: "center",
+    padding: theme.spacing(1),
+    borderRadius: "10%",
+    "&:hover": {
+      backgroundColor: "#029a82",
+    },
+  },
+  rejectButton: {
+    textAlign: "center",
+  },
+  emailMessage: {
+    width: "100%",
+  },
+}));
+
+const JobPostDetails = () => {
+  const classes = useStyles();
+  //const params = useParams();
+  const params = "60bcffddc9068f6473ec271f";
+  const [openEmail, setOpenEmail] = useState(false);
+
+  const jobData = jobs.filter((post) => post._id.$oid === params)[0];
+  //console.log(jobData);
+  let candidates = jobApplicantsForPost.filter(
+    (el) => el.jobPostId === params
+  )[0].candidates;
+  console.log(candidates);
```

Comment:
Remove these consol logs

Reply:
sure.. forgot to comment those console logs because it was late night.. yes now starting the api